apps/khix/src/app/_components/sections/hero/Hero.module.css

```
@@ -0,0 +1,384 @@
+  background:
+    linear-gradient(
+      180deg,
+      rgba(247, 253, 213, 0.96),
+      rgba(219, 240, 226, 0.9)
+    ),
+    #eef6cf;
+}
+
+.art {
+  --khix-art-motion-scale: 0.64;
+  position: absolute;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)